bindings/go/go_motr_api_server/README.md

```diff
@@ -0,0 +1,67 @@
+
+## Motr REST API server
+
+Motr REST API Server is a simple REST API Server which based on motr go bindings. 
+
+It included a REST API server and a http client.
+
+The API implemented as below:
+
+```
+/api/kv/put
+/api/kv/get
+/api/kv/delete
+/api/object/read
+/api/object/write
```

Comment:
What is `/api/`?

Reply:
This is just part of the path of the url, should I remove it so the url will be much more simple ?

Reply:
Yes, please.

Reply:
Sorry for late reply, I just back from vacation, I will remove the "/api" path accrodingly.

Reply:
I think the leading `/` is not needed also.

Reply:
`/` removed.